CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This file is used to list changes made in each version of the apt cookbook.
 
+## 7.0.1 (2018-09-05)
```

Comment:
In the future we have tooling that writes these out automatically so we make sure the dates line up with the date we release the cookbook.